publishing-rules.yaml

```diff
@@ -0,0 +1,68 @@
+skip-godeps: true
+rules:
+- destination: kubernetes
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/kubernetes
+- destination: kubernetes-gengo
+  branches:
+  - name: openshift-3.9
+    source:
+      branch: master
+      dir: vendor/k8s.io/gengo
+- destination: google-certificate-transparency
+  branches:
+  - name: master
+    source:
+      branch: master
```

Comment:
I think we're going to want to use the "normal" branch we already have or one very similarly named.

Reply:
change to "openshift-3.9" will have to be created